lighthouse-core/test/audits/exists.js

```diff
@@ -41,4 +65,17 @@ describe('Manifest: exists audit', () => {
       }
     }).debugString, debugString);
   });
+
+  it('correctly passes through a JSON parsing failure', () => {
+    const artifacts = {
+      Manifest: manifestParser('{ \name: Definitely not valid JSON }')
+    };
+    const output = Audit.audit(artifacts);
+    assert.equal(output.rawValue, false);
+    assert.ok(output.debugString.includes('Unexpected token'), 'No JSON error message');
```

Comment:
No idea. Probably not spec'd but i'm sure such a v8 change will break a lot. So i'm okay if it also breaks us. :)